src/reconcile/rec_visibility.c

```diff
@@ -644,7 +644,20 @@ __rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_UPDATE *first_upd
             }
         }
 
-        /* Always select the newest committed update to write to disk */
```

Comment:
Do we want to keep this old comment for the `upd_select->upd` test later?

Reply:
I don't understand why?